geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/ExportCommandTools.scala

```diff
@@ -0,0 +1,90 @@
+/***********************************************************************
+  * Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+  * All rights reserved. This program and the accompanying materials
+  * are made available under the terms of the Apache License, Version 2.0
+  * which accompanies this distribution and is available at
+  * http://www.opensource.org/licenses/apache2.0.php.
+  *************************************************************************/
+
+package org.locationtech.geomesa.tools.accumulo.commands
+
+import java.io._
+import java.util.zip.GZIPOutputStream
+import scala.collection.JavaConversions._
+
+import com.beust.jcommander.{Parameter, ParameterException}
+import com.typesafe.scalalogging.LazyLogging
+import org.geotools.data.Query
+import org.geotools.data.simple.SimpleFeatureCollection
+import org.geotools.filter.text.ecql.ECQL
+import org.locationtech.geomesa.accumulo.data.{AccumuloDataStore, AccumuloFeatureStore}
+import org.locationtech.geomesa.tools.accumulo._
+import org.locationtech.geomesa.tools.accumulo.Utils.setOverrideAttributes
+import org.locationtech.geomesa.tools.common.{FeatureTypeNameParam, OptionalCQLFilterParam}
+import org.opengis.filter.Filter
+
+import scala.util.{Failure, Success, Try}
+
+trait ExportCommandTools[P <: ExportCommandToolsParam] extends LazyLogging {
```

Comment:
I don't think separating out the export command stuff here was really necessary, but it's fine I suppose.